src/join/composer.jl

```diff
@@ -485,7 +503,8 @@ change in future releases.
   data frame and left unchanged.
 - `matchmissing` : if equal to `:error` throw an error if `missing` is present
   in `on` columns; if equal to `:equal` then `missing` is allowed and missings are
-  matched (`isequal` is used for comparisons of rows for equality)
+  matched; if equal to `:notequal` then missings are dropped in `df1` and `df2` 
+  `on` columns (`isequal` is used for comparisons of rows for equality)
```

Comment:
```suggestion
  `on` columns (in the two latter cases `isequal` is used for comparisons of rows for equality)
```
(and same below)

Reply:
`isequal` is used in all cases to test for equality so I think what we had before was correct; maybe just it should be a stand alone sentence - not something put in parentheses